cpp/src/column/column.cu

```diff
@@ -123,7 +120,7 @@ mutable_column_view column::mutable_view() {
                              size(),
                              _data.data(),
                              static_cast<bitmask_type *>(_null_mask.data()),
-                             current_null_count,
+                             cudf::UNKNOWN_NULL_COUNT,
```

Comment:
I think it makes sense to use the internal null count here. That way if it's known, it's available w/o a kernel. 
```suggestion
                             _null_count,
```

Reply:
Also, add a doc about why we're making this change. Something about how a mutable view is always going to be used to mutate the column, and therefore it's usually unnecessary to compute the null count since it's likely going to be modified.

Reply:
Duh! I hadn't considered that this method has access to all the `column` internals. Perfect!